src/fact_generator.ts

```diff
@@ -452,3 +476,38 @@ async function mostGaonAppearances(): Promise<string[]> {
         .limit(25);
     return result.map((x, idx) => `Fun Fact: '${x.artist_name}' has placed on the GAON digital weekly charts the ${getOrdinalNum(idx + 1)} most times with ${x.appearances} appearances!`);
 }
+
+async function historicalGaonWeekly(): Promise<Array<string>> {
+    const startYear = 2010;
+    const endYear = new Date().getFullYear() - 1;
```

Comment:
Can you explain why you're doing -1 here and +1 on line 489? I don't get it

Reply:
These quotes show songs for previous years, `endYear` evaluates to 2019 in 2020. Line 489 generates a range from 2010 to 2019, and the +1 creates the correct array size otherwise you'd have an off by one error